feat(perf): Add span details histogram note for EA (#4960)
getsentry/sentry-docs: src/docs/product/performance/transaction-summary.mdx

Choose from one of several [metrics](#metrics) to sort spans and identify different types of problems. Filter spans to only see the span operation type you're interested in.

> Clicking on a span will take you to its summary page that should give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.

Reviewer: ```suggestion
Clicking on a span will take you to its summary page that will give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
```

I feel like it's better to assume it will give more details than leave the user wondering whether it will or won't.

> Clicking on a span will take you to its summary page that will give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
> 
> <Note>
> 
> If you're in the Early Adopter program, you can also see the distribution of the span self time. This histogram can help you identify systemic performance issues across the entire dataset by showing how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the set bounds. Adding filters allows you to focus on the best or worse performing spans so you can troubleshoot performance issues more easily.
> 

Reviewer 2: ```suggestion

### Span Summary

Clicking on a span will take you to its summary page which will give you more details about this specific span [group](#grouping). On the page, you can see the performance of the span over time and see a list of transaction events that contain the span.

<Note>

This span distribution feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

If you're in the Early Adopter program, you can also see the distribution of the span self time using the (added) "Display" dropdown. This histogram can help you identify systemic performance issues across the entire dataset by showing how self time for every span instance is distributed. 

You can also select a subset of the dataset by zooming into the distribution histogram. The stats and transaction events table will reflect that change and will only include spans with self time that are within the bounds of your subset. This allows you to focus on the best or worse performing spans so you can troubleshoot performance issues more easily.

```

> </Note>

Reviewer 2: ```suggestion
```

> If you're in the Early Adopter program, you can also see the distribution of the span self time. The self time histogram can help you identify systemic performance issues accross the entire dataset by illustrating how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max self time bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the specified bounds. Adding self time filters allows you to narrow down your focus on the best/worse performing spans so you can troubleshoot performance issues easier.

Reviewer: I feel like self time needs to be hyphenated (so does Grammarly)

Author: It looks like in the product, we're referring to it as [`self time`](url). I wonder if we can make an exception since it's a technical term? I don't hold a strong opinion though, just flagging that we'll need to update all other self time mentions in the [docs](http://localhost:3000/product/performance/transaction-summary/#self-time) and in the product

Reviewer: Sounds good. I'm fine with that, just wanted to make sure.

Reviewer 2: Also prefer we're consistent with the product. If we change it, we change it everywhere, but generally, we should quote the UI.

> If you're in the Early Adopter program, you can also see the distribution of the span self time. The self time histogram can help you identify systemic performance issues accross the entire dataset by illustrating how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max self time bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the specified bounds. Adding self time filters allows you to narrow down your focus on the best/worse performing spans so you can troubleshoot performance issues easier.

Reviewer: ```suggestion
If you're in the Early Adopter program, you can also see the distribution of the span self time. The histogram can help you identify systemic performance issues across the entire dataset by illustrating how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the specified bounds. Adding filters allows you to narrow down your focus on the best/worse performing spans so you can troubleshoot performance issues easier.
```

I feel like we've said self time (self-time?) a lot here. I removed a couple of instances but may need to play around with where we include it.

Author: This reads better for sure. Yeah I've never written self time so much in a single paragraph 😅
